src/dmd/target.h

```diff
@@ -86,4 +86,6 @@ struct Target
     static TypeTuple *toArgTypes(Type *t);
     static bool isReturnOnStack(TypeFunction *tf, bool needsThis);
     static d_uns64 parameterSize(const Loc& loc, Type *t);
+    static Expression getTargetInfo(const char* name, const Loc& loc);
+    static const char *const * allTargetInfos(size_t* numTargetInfos);
```

Comment:
Are you sure you checked the signature is correct here?

Reply:
Yes: `extern (C++) static const(char*)* allTargetInfos(size_t* numTargetInfos)`
Is there a problem?

Reply:
This use of const mangles correctly. It's not a head-const.
Do you think this PR will survive?